flux_led/protocol.py

```diff
@@ -76,9 +79,9 @@ class LevelWriteMode(Enum):
 #     |  |  |  |  |  |  |  |  |  |  |  |  |  |
 #     |  |  |  |  |  |  |  |  |  |  |  |  |  checksum
 #     |  |  |  |  |  |  |  |  |  |  |  |  color mode (f0 colors were set, 0f whites, 00 all were set)
-#     |  |  |  |  |  |  |  |  |  |  |  cool-white  0x00 to 0xFF
+#     |  |  |  |  |  |  |  |  |  |  |  warm-white  0x00 to 0xFF
 #     |  |  |  |  |  |  |  |  |  |  version number
-#     |  |  |  |  |  |  |  |  |  warmwhite  0x00 to 0xFF
+#     |  |  |  |  |  |  |  |  |  cool-white  0x00 to 0xFF
```

Comment:
Maybe this is right, but that would be weird since the original protocol only had one white slot and most RGBW strips are cool white